var/spack/repos/builtin/packages/mpich/package.py

```diff
@@ -471,6 +475,11 @@ def configure_args(self):
         elif 'pmi=cray' in spec:
             config_args.append('--with-pmi=cray')
 
+        if '+cuda' in spec:
+            config_args.append('--with-cuda={0}'.format(spec['cuda'].prefix))
+        else:
+            config_args.append('--without-cuda')
```

Comment:
```suggestion
        config_args += self.with_or_without('cuda', activation_value='prefix')
```

is equivalent, but whatever you find more readable :D

Reply:
Thanks, I adopted this as suggested.